activities/Our Performance.txt

```diff
@@ -0,0 +1 @@
+We have been holding our annual concert since 2011, having the main purpose of showcasing angklung and introducing NUS Angklung Ensemble to the NUS community. In our 2015 concert, “The Chronicle”, we performed 13 well-known songs spanning diverse eras to take the audience in a journey through time. As the biggest concert in the history of NUS Angklung Ensemble, “The Chronicle” successfully attracted 150 people, including the Education Attaché of the Embassy of the Republic of Indonesia in Singapore, Mr. Ismunandar, and the manager of NUS Office of Student Affairs, Ms. Lyana Wang Wanzhen.
```

Comment:
You might want to clear the raw txt file since you know have content javascript.